app/styles/base.css

```diff
@@ -654,6 +654,79 @@ select:active {
 }
 
 /* ----------------------------------------
+ * Loading Animation
+ * ----------------------------------------
+ */
+
+#noVNC_loading_anim {
```

Comment:
`noVNC_loading` would be new, but we have a `noVNC_transition`. This diff is what I did:

```diff
diff --git a/app/styles/base.css b/app/styles/base.css
index d166bb3..087089e 100644
--- a/app/styles/base.css
+++ b/app/styles/base.css
@@ -769,6 +769,7 @@ select:active {
   justify-content: center;
   flex-direction: column;
 }
+:root.noVNC_loading #noVNC_transition,
 :root.noVNC_connecting #noVNC_transition,
 :root.noVNC_disconnecting #noVNC_transition,
 :root.noVNC_reconnecting #noVNC_transition {
diff --git a/app/ui.js b/app/ui.js
index 98bb5f6..db4f167 100644
--- a/app/ui.js
+++ b/app/ui.js
@@ -170,6 +170,8 @@ var UI;
 
             document.getElementById('noVNC_setting_host').focus();
 
+            document.documentElement.classList.remove("noVNC_loading");
+
             var autoconnect = WebUtil.getConfigVar('autoconnect', false);
             if (autoconnect === 'true' || autoconnect == '1') {
                 autoconnect = true;
diff --git a/vnc.html b/vnc.html
index c6f0d25..de4d5bb 100644
--- a/vnc.html
+++ b/vnc.html
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html class="noVNC_loading">
 <head>
 
     <!--
```

Very minimal. :)